src/test/java/net/sf/jabref/importer/fileformat/BibTeXMLImporterTest.java

```diff
@@ -40,9 +40,10 @@
 
     }
 
-    @BeforeClass
-    public static void setUp() {
+    @Before
+    public void setUp() throws Exception {
         Globals.prefs = JabRefPreferences.getInstance();
+        importer = new BibTeXMLImporter();
```

Comment:
Add this as separate `resetImporter` with `@Before`  and keep the old `@BeforeClass`.  Reason: The `@BeforeClass` usage is a pattern described at https://github.com/JabRef/jabref/wiki/Code-Howtos#preferences-in-tests.